### PR TITLE
Update to Electron 1.2.8

### PR DESCRIPTION
**Pull request by miniak**
case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

File: github-electron/github-electron.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for Electron v1.2.7
+// Type definitions for Electron v1.2.8
```
Comment (rhysd) [inline github-electron/github-electron.d.ts]: Electron v1.3.0 was released without any change to APIs so should we mark as v1.3.0 here?

**Comment by rhysd** [comment]
Sorry for late response 💦 

Changes look good except for above comment 👍  Please check it.

**Comment by miniak** [comment]
@rhysd I would keep the version, does not really matter.

**Comment by rhysd** [comment]
@miniak  

That's also OK :+1:

@vvakame 

LGTM :+1:

**Comment by yuit** [comment]
@miniak  thanks for PR and @rhysd thanks for reviewing

**Pull request merged**